src/main/kotlin/graphics/scenery/controls/RangeSlider.kt

```diff
@@ -0,0 +1,99 @@
+package graphics.scenery.controls
```

Comment:
The `control` package contains the low-level stuff, like button handling, etc. I'd argue this rather belongs into a new package, e.g. named `ui` 👍